tests/NumberTest.php

```diff
@@ -14,19 +14,20 @@ public function it_has_attributes($number, $decimal, $half, $currentEven, $negat
     {
         $number = Number::fromString($number);
 
-        $this->assertEquals($decimal, $number->isDecimal());
-        $this->assertEquals($half, $number->isHalf());
-        $this->assertEquals($currentEven, $number->isCurrentEven());
-        $this->assertEquals($negative, $number->isNegative());
-        $this->assertEquals($integerPart, $number->getIntegerPart());
-        $this->assertEquals($fractionalPart, $number->getFractionalPart());
-        $this->assertEquals($negative ? '-1' : '1', $number->getIntegerRoundingMultiplier());
+        $this->assertSame($decimal, $number->isDecimal());
```

Comment:
Actually I do. It's because of the scale variable with BC math. Then trailing zero's are added. So we are not using `assertSame` in calculator tests.